xss_utils/templatetags/README.rst

```diff
@@ -0,0 +1 @@
+- Contains custom tags for Django Templates to prevent XSS.
```

Comment:
Can you also add a link to the wiki page with text that reads something like: "For more detailed instructions, see preventing XSS in Django Templates here.", and link `preventing XSS in Django Templates here` to the wiki.